dist/module.js

```diff
@@ -0,0 +1,64 @@
+// *** Imports ***
+// Node.js dependencies
+const fs = require('fs');
+const md5 = require('md5');
+const path = require('path');
+
+// *** File compression configuration ***
+const temporaryDirectoryPrefix = 'temp';
+const compressionCommand = 'tar -zcf';
+const expandCommand = 'tar -zxf';
+const compressedFileExtension = '.tar.gz';
+
+function fileNameForCache(key, cachePath) {
+  return [key, '_', md5(cachePath), compressedFileExtension].join('');
+}
+
+function createFolder(folderPath) {
+  // Equivalent UNIX command: await exec.exec('mkdir -p', [folderPath], { silent: true });
+
+  if (fs.existsSync(folderPath)) { return };
+  fs.mkdirSync(folderPath, { recursive: true });
+}
+
+function deleteFolder(folderPath) {
+  // Equivalent UNIX command: await exec.exec('rm -rf', [folderPath], { silent: true });
+  // From: https://stackoverflow.com/a/32197381/13170523
+
+  if (!fs.existsSync(folderPath)) { return };
+
+  fs.readdirSync(folderPath).forEach((file, index) => {
+    const filePath = path.join(folderPath, file);
+    if (fs.lstatSync(filePath).isDirectory()) {
+      // Delete path recursively
+      deleteFolder(filePath);
+    } else {
+      // Delete file
+      fs.unlinkSync(filePath);
+    }
+  });
+
+  fs.rmdirSync(folderPath);
+}
+
+function moveFolderContents(originFolderPath, targetFolderPath) {
+  // TODO: implement it
+}
+
+module.exports = {
+  // Module imports (to avoid having to import them again)
+  path: path,
+  fs: fs,
+
+  // Module constants
+  compressedFileExtension: compressedFileExtension,
+  temporaryDirectoryPrefix: temporaryDirectoryPrefix,
+  compressionCommand: compressionCommand,
+  expandCommand: expandCommand,
+
+  // Module functions
+  fileNameForCache: fileNameForCache,
+  createFolder: createFolder,
+  deleteFolder: deleteFolder,
+  moveFolderContents: moveFolderContents,
```

Comment:
use shorthand syntax
```suggestion
  fileNameForCache,
  createFolder,
  deleteFolder,
  moveFolderContents,
```